### ISSUE TITLE
How to get the assigned ip in the ippool

### ISSUE THREAD

**Issue by lliu0947** [projectcalico/calico#7046]
libcalico-go is intended for internal use only,  [github.com/projectcalico/api](https://github.com/projectcalico/api) for the officially supported API definition and golang clients. but there is no ipam related api. I want to get the IP assigned by the IPAM. What can I do now，do you have plans to provide such apis.

**Comment by lliu0947** [comment]
> The IPAM API is not public and is intended for internal use only.
> 
> What is your use-case?

I want to know which ip has been allocated to a certain ip pool, and determine whether the ippool is in use（Use calicoctl ipam to solve）

**Comment by caseydavenport** [comment]
Yep, the way to do that right now is with the `calicoctl ipam` command.

We do also have read-only access for IPAM BlockAffinities, via the k8s API, but that won't get you individual IP addresses for now.
